### ISSUE TITLE
Monospaced Variant

### ISSUE THREAD

**Issue by svictoreq** [crozynski/comicneue#10]
The font is beautiful. I really like the work you've done. So, I think it really would be nice to have a monospaced variant for programmers.

**Comment by aolko** [comment]
pardon, why? You think that your code should be funny?

**Comment by svictoreq** [comment]
Yeah, I actually like the font. I tried it in my editor, but since it's not monospaced, the code loses readability.

**Comment by kilobyte** [comment]
Converting a proportional font into monospaced is non-trivial amount of work.  Try for example placing "lllllll" or "iiiiiii" at equally spaced positions, then do the same for "mmmmm".  It'd take redesigning quite a few letters to produce a reasonable result.

**Comment by thymaro** [comment]
on another note, the font is not hinted and probably won't look extremely readable in a programming environment, at least not as good as on paper. I mean it wouldn't be beneficial for the eyes. Consider using a manually hinted teletype font for your programming ;)

**Comment by dwelle** [comment]
There's [shannpersand/comic-shanns](https://github.com/shannpersand/comic-shanns) and it looks great as is, so I believe Comic Neue would look great as well.

**Comment by APerricone** [comment]
I want use this font in a [boxes input](https://stackoverflow.com/questions/21131678/seperate-boxes-for-each-letter-in-an-input-field) so a monospace version would be great

**Comment by DonaldTsang** [comment]
https://github.com/shannpersand/comic-shanns

**Comment by davewongillies** [comment]
> https://github.com/shannpersand/comic-shanns

And also [comic mono](https://github.com/dtinth/comic-mono-font/)
